### DISCUSSION TITLE
QC about TSS.enrichment and nucleosome_signal

### DISCUSSION THREAD

**Discussion by siqi-zZ** [stuart-lab/signac#1742] [Q&A] [answered]
Hello,

I have a question, I found our sample have low scATAC-seq library quality.  can I change filter value?
like stander is 
pbmc <- subset(
  x = pbmc,
  subset = nCount_peaks > 3000 &
    nCount_peaks < 30000 &
    pct_reads_in_peaks > 15 &
    blacklist_ratio < 0.05 &
    nucleosome_signal < 4 &
    TSS.enrichment > 3
)
pbmc
### 
can I change like this for got more fregments?
pbmc_donor1 <- subset(
  x = pbmc_donor1,
  subset = nCount_peaks > 3000 &
    nCount_peaks < 50000 &
    pct_reads_in_peaks > 15 &
    blacklist_ratio < 0.05 &
    nucleosome_signal < 1 &     # nucleosome_signal < 4 &
    TSS.enrichment > 1          # TSS.enrichment > 3    
)
The attachement is my sample before QC.
Wati for respond and thinks for useful package!!!!
<img width="333" alt="image" src="https://github.com/stuart-lab/signac/assets/21080145/d715a23f-f7c2-4859-9506-54a244a5a5ab">

**Comment by timoast** [comment, marked as answer]
Yes, these thresholds are meant to be adjusted for each dataset. The information shown on the website is only a guide.

**Reply by siqi-zZ** [reply]
thank you so much!!
